app/containers/Switch/index.tsx

```diff
@@ -7,11 +7,11 @@ const Switch = (props: SwitchProps): React.ReactElement => {
 	const { colors } = useTheme();
 
 	const trackColor = {
-		false: colors.buttonBackgroundDangerPress,
+		false: colors.fontDisabled,
 		true: colors.buttonBackgroundSuccessPress
```

Comment:
```suggestion
		false: colors.strokeDark,
		true: colors.buttonBackgroundPrimaryDefault
```
I read your comment about the color, and we can slightly change the order of the colors to get the correct result. I'll add the correct version here.